daemon/api.go

```diff
@@ -257,6 +258,12 @@ var (
 		POST:     ackWarnings,
 	}
 
+	modelCmd = &Command{
```

Comment:
if possible it would be best to follow the new pattern and not add to this file, but have api_model*.go files. see recent addition

Reply:
Done now by John (thank you!)